### PR TITLE
v1.26 Add/FriendliAI

### PR DESCRIPTION
**Pull request by snowman39**
### What's being changed:

Add FriendliAI module integration pages.

We made this PR following the existing practice and adding the content we would like to introduce to the integration page.
Please feel free to share any feedback you have!

### Type of change:

- [x] **Documentation** updates (non-breaking change to fix/update documentation)

### How Has This Been Tested?

- [ ] **GitHub action** – automated build completed without errors
- [x] **Local build** - the site works as expected when running `yarn start`

> note, you can run `yarn verify-links` to test site links locally

### CODE REVIEW

**Comment by snowman39** [comment]
I agree with CLA

File: developers/weaviate/model-providers/_includes/provider.connect.py [hunk 1]
```diff
@@ -79,6 +87,13 @@
 # START CohereInstantiation
     "X-Cohere-Api-Key": cohere_key,
 # END CohereInstantiation
+# START FriendliInstantiation
+    "X-Friendli-Api-Key": friendli_key,
+# END FriendliInstantiation
+# START FriendliDedicatedInstantiation
+    "X-Friendli-Api-Key": friendli_key,
+    "X-Friendli-Baseurl": "https://inference.friendli.ai/dedicated",
```
Comment (antas-marcin) [inline developers/weaviate/model-providers/_includes/provider.connect.py:95]: default base url defined in Weaviate module is `https://inference.friendli.ai` here is different, is this OK?

Comment (snowman39) [reply developers/weaviate/model-providers/_includes/provider.connect.py:95]: This is OK. 
This is right configuration for users who want to use their own models deployed on Friendli Suite :)

**Comment by databyjp** [comment]
Hi @snowman39 - thank you for this, it is great. I have some minor tweaks to make, but I will make them after merging and let you know FYI :)

**Comment by databyjp** [comment]
Hi @snowman39 :) 
I have made these relatively minor changes, mostly for consistency with the rest of the docs. Please let me know if there are any issues.
https://github.com/weaviate/weaviate-io/pull/2560

**Pull request merged**